README.md

```diff
@@ -210,6 +210,22 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
   > **NOTE** ⏳ This command can take few minutes or immediately depending on your internet connection or model is cached. If you have issues connecting to Hugging Face, refer to the [Hugging Face discussion forum](https://discuss.huggingface.co/) for more details.
 
+  #### Downloading a specific model from a Hugging Face repository
+
+- Specify repository, model, and a Hugging Face token if necessary
```

Comment:
Can we point users to the HF doc about tokens? https://huggingface.co/docs/hub/en/security-tokens